public/app.js

```diff
@@ -9,6 +9,9 @@
  *
  * Find more information about this on the LICENSE file.
  */
+import { checkPluginVersion } from "./utils";
+checkPluginVersion();
```

Comment:
Move the `checkPluginVersion()`'s function execution to `app.run` callback

Reply:
I added a "try/catch" block to avoid errors in the application.

It is a pre-check to clean the data browser before the application starts, I think it should be executed as soon as possible to avoid the "health-check" to be executed in parallel.

Reply:
I moved the `checkPluginVersion` method into the `app.run` method and it works fine.